app/Component/Datagrid/Datagrid.js

```diff
@@ -44,10 +45,10 @@ class Datagrid extends React.Component {
     }
 
     buildHeaders() {
-        let headers = [];
-
-        let sortDir = this.state.data.get('sortDir');
-        let sortField = this.state.data.get('sortField');
+        let headers = [],
+            actions = this.props.view.listActions(),
+            sortDir = this.state.data.get('sortDir'),
+            sortField = this.state.data.get('sortField');
```

Comment:
I prefer one `let` per line. It's really not a real optimization and way harder to read diffs.